lib/apiservers/engine/backends/container.go

```diff
@@ -818,7 +841,7 @@ func attachStreams(name string, tty, stdinOnce bool, clStdin io.ReadCloser, clSt
 				// behavior where you connect to stdin on the first time only.
 				// If we really want to add this behavior, we need to add support
 				// in the ssh tether in the portlayer.
-				log.Printf("Attach stream has stdinOnce set.  VIC does not yet support this.")
+				log.Debugf("Attach stream has stdinOnce set.  VIC does not yet support this.")
```

Comment:
This should be Warnf or Errorf - could be significant if attempting to diagnose behavioural issue for customer.